gpkit/constraints/bounded.py

```diff
@@ -94,10 +95,10 @@ def process_result(self, result):
                 out["value near upper bound"].append(varkey)
         if self.verbosity > 0 and out:
             print
-            print "UNBOUNDED VARIABLES"
+            print "Feasible with these variables bounded:"
```

Comment:
This header is declaring that the model solved under some combination of relaxation and bounding; the contents after the header declare exactly what was done.